doc/design/reader/README.md

```diff
@@ -1,35 +1,35 @@
-# Python Data Reader Design Doc
+# Python Data Reading Design Doc
 
-Paddle reads data from data reader during training. It will be passed into `paddle.train` as a parameter.
+Paddle reads data from *data reader* during training. *data reader creator* (or *reader creator*) creates a *data reader* when invoked. *reader creator* will be passed into `paddle.train` as a parameter.
 
-## Data Reader Interface
+## Data Reader Creator Interface
 
-Data reader is a function with no parameter that creates a iterable (anything can be used in `for x in iterable`):
+Data reader creator is a function with no parameter that creates a iterable (anything can be used in `for x in iterable`):
```

Comment:
I think a reader creator can have parameters. For example:

```python
def reader_creator_text(filename):
    def reader():
        ... read from filename ...
    return reader
```

Here the creator does have one paraemter.